src/plugins/jartic/datasource.py

```
@@ -0,0 +1,268 @@
+    ParameterType,
+    Sensor,
+)
+from .archive_downloader import JARTICArchiveDownloader
+from .data_parser import JARTICDataParser
+
+
+logger = logging.getLogger(__name__)
+
+
+class JARTICDataSource(DataSource):
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)